packages/odata-v2/src/request-builder/get-all-request-builder.spec.ts

```diff
@@ -229,9 +230,14 @@ describe('GetAllRequestBuilder', () => {
       );
 
       try {
-        await requestBuilder.top(1).timeout(10).execute(defaultDestination);
+        await requestBuilder
+          .top(1)
+          .middleware([timeout(10)])
+          .execute(defaultDestination);
       } catch (err) {
-        expect(err.cause.message).toBe('timeout of 10ms exceeded');
+        expect(err.message).toBe(
```

Comment:
[q] Is this intended? Removing `.cause` could break something, right?

Reply:
In the past the error was coming from axios leading a error with cause. Now the exception is created by us leading to a exception without a cause.